lua/lsp/python.lua

```diff
@@ -74,6 +74,7 @@ M.lsp = function()
         update_in_insert = true,
       }),
     },
+    root_dir = require("lspconfig").util.root_pattern(".git/", "requirements.txt"),
```

Comment:
the default value of this is (as described [here](https://github.com/neovim/nvim-lspconfig/blob/master/CONFIG.md#pyright)):
```lua
root_dir = function(filename)
    return util.root_pattern(unpack(root_files))(filename) or util.path.dirname(filename)
end,
```
that doesn't work for you?

Reply:
Good point @abzcoding ! I did not consider the default value.
The default value seems to be enough on pyright. The project root (i.e. root_dir) was resolved correctly without explicitly specifying the root_dir. Before I made the PR there was some issue with one of my Python projects project root detection, but I was not able to reproduce it. It must not been related to the root_dir after all. 

I will close the PR now as the default value for root_dir seems to be fine.